beacon-chain/rpc/prysm/v1alpha1/validator/attester_test.go

```diff
@@ -201,94 +200,6 @@ func TestGetAttestationData_Optimistic(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestAttestationDataAtSlot_HandlesFarAwayJustifiedEpoch(t *testing.T) {
```

Comment:
Moved to `beacon-chain/rpc/prysm/v1alpha1/validator/test-mainnet/attester_test.go`